cli/include/vendored-cli/ignition/utils/cli/CLI.hpp

```diff
@@ -1,36 +1,18 @@
-// Copyright (c) 2017-2020, University of Cincinnati, developed by Henry Schreiner
-// under NSF AWARD 1414736 and by the respective contributors.
-// All rights reserved.
-//
-// SPDX-License-Identifier: BSD-3-Clause
-
-#pragma once
-
-// CLI Library includes
-// Order is important for combiner script
-
-#include "Version.hpp"
-
-#include "Macros.hpp"
-
-#include "StringTools.hpp"
-
-#include "Error.hpp"
-
-#include "TypeTools.hpp"
-
-#include "Split.hpp"
-
-#include "ConfigFwd.hpp"
-
-#include "Validators.hpp"
-
-#include "FormatterFwd.hpp"
-
-#include "Option.hpp"
-
-#include "App.hpp"
-
-#include "Config.hpp"
-
-#include "Formatter.hpp"
+/*
+ * Copyright (C) 2022 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#include <vendored-cli/gz/utils/cli/CLI.hpp>
```

Comment:
`vendored-cli/` should be removed from all the `#include`s in these files

Reply:
[gitkraken://repolink/04e75deaed05c1de65090b0b0174012610350bb7/commit/f3c7fcdc2ab59e50a8ea105845feaa57b3f221ed?url=git%40github.com%3Aignitionrobotics%2Fign-utils.git](https://github.com/ignitionrobotics/ign-utils/pull/55/commits/f3c7fcdc2ab59e50a8ea105845feaa57b3f221ed)